tests/test_optimizely.py

```diff
@@ -266,6 +266,19 @@ def test_init__sdk_key_and_datafile_access_token(self):
 
         self.assertIs(type(opt_obj.config_manager), config_manager.AuthDatafilePollingConfigManager)
 
+    def test_init__invalid_default_decide_options(self):
```

Comment:
what about moving this test to "test_user_context" along with other tests for defaut_decide_options?

Reply:
Sure, that makes sense. I can include that in my next PR I am working on for the Events and Attributes addition so that should be coming shortly.

Reply:
Moved over decide tests to test_user_context.